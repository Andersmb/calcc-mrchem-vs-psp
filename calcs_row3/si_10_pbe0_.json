{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -1.0353884783316283
          ],
          "radius": 1.46
        },
        {
          "center": [
            0.0,
            0.0,
            1.817530851668372
          ],
          "radius": 0.65
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "si",
          "xyz": [
            0.0,
            0.0,
            -1.0353884783316283
          ]
        },
        {
          "atom": "o",
          "xyz": [
            0.0,
            0.0,
            1.817530851668372
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "si_10_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        1.0108673694733299e-16
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.2385497407663202,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            6.136334026591383e-07,
            6.389901924215022e-06,
            -1.2385497407496848
          ],
          "vector_el": [
            6.135108451586988e-07,
            6.389779325568577e-06,
            -1.28335785745373
          ],
          "vector_nuc": [
            1.225575004394073e-10,
            1.2259864644490868e-10,
            0.044808116704045275
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Si",
          "xyz": [
            0.0,
            0.0,
            -1.0353884783316283
          ]
        },
        {
          "symbol": "O",
          "xyz": [
            0.0,
            0.0,
            1.817530851668372
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -66.28124526504837,
          -19.192113319101917,
          -5.3648881961069765,
          -3.699858054628485,
          -3.6998574137223903,
          -3.69311292621859,
          -0.969410212384127,
          -0.4535102898401226,
          -0.3491221557367352,
          -0.34912106065005977,
          -0.32437725871996126,
          -66.28124527507981,
          -19.192113300664285,
          -5.364888177470231,
          -3.6998580506095764,
          -3.6998574326769433,
          -3.6931129228763884,
          -0.9694101800034164,
          -0.453510291117046,
          -0.34912222258379744,
          -0.34912091774734355,
          -0.32437724639092624
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -208.7532321693775
      },
      "scf_energy": {
        "E_ee": 208.62856557153745,
        "E_eext": 0.0,
        "E_el": -403.8327887010207,
        "E_en": -947.0924845456389,
        "E_kin": 363.88383341367864,
        "E_next": 0.0,
        "E_nn": 39.25803257815915,
        "E_nuc": 39.25803257815915,
        "E_tot": -364.5747561228616,
        "E_x": -7.129334736281606,
        "E_xc": -22.123368404316384,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 209.77136206549977,
        "E_eext": 0.0,
        "E_el": -401.71014435491566,
        "E_en": -944.2234304826219,
        "E_kin": 362.17535353467144,
        "E_next": 0.0,
        "E_nn": 39.25803257815915,
        "E_nuc": 39.25803257815915,
        "E_tot": -362.4521117767565,
        "E_x": -7.1824340721545905,
        "E_xc": -22.25099540031036,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 205.93646231558915,
              "E_eext": 0.0,
              "E_el": -403.68925564623163,
              "E_en": -939.5492441996629,
              "E_kin": 358.85157906177494,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.4312230680725,
              "E_x": -7.050150978379832,
              "E_xc": -21.877901845553048,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.4312230680725,
            "energy_update": 1.9791112913159736,
            "mo_residual": 0.464021309217895,
            "wall_time": 87.22341189
          },
          {
            "energy_terms": {
              "E_ee": 208.93354159733806,
              "E_eext": 0.0,
              "E_el": -403.80250817357216,
              "E_en": -946.0496506312171,
              "E_kin": 362.5951655925417,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.544475595413,
              "E_x": -7.138071246469791,
              "E_xc": -22.143493485765102,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.544475595413,
            "energy_update": 0.11325252734053493,
            "mo_residual": 0.27593869855680797,
            "wall_time": 63.310133119
          },
          {
            "energy_terms": {
              "E_ee": 207.95064920809241,
              "E_eext": 0.0,
              "E_el": -403.81910688760985,
              "E_en": -945.431845329757,
              "E_kin": 362.8333901663436,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5610743094507,
              "E_x": -7.108867176689336,
              "E_xc": -22.062433755599358,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5610743094507,
            "energy_update": 0.016598714037684204,
            "mo_residual": 0.16262998685158792,
            "wall_time": 54.185087164
          },
          {
            "energy_terms": {
              "E_ee": 208.47583021043508,
              "E_eext": 0.0,
              "E_el": -403.8298604495403,
              "E_en": -946.4433589339095,
              "E_kin": 363.37086084837483,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5718278713812,
              "E_x": -7.12457528640741,
              "E_xc": -22.10861728803332,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5718278713812,
            "energy_update": 0.010753561930471278,
            "mo_residual": 0.07538805417486708,
            "wall_time": 53.033869261
          },
          {
            "energy_terms": {
              "E_ee": 208.65817021965748,
              "E_eext": 0.0,
              "E_el": -403.83211716507344,
              "E_en": -947.1311341873537,
              "E_kin": 363.8970333384997,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5740845869143,
              "E_x": -7.129947852059113,
              "E_xc": -22.12623868381772,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5740845869143,
            "energy_update": 0.002256715533121678,
            "mo_residual": 0.03161746484060194,
            "wall_time": 50.93084475
          },
          {
            "energy_terms": {
              "E_ee": 208.66408740206836,
              "E_eext": 0.0,
              "E_el": -403.8326236231467,
              "E_en": -947.2985779648222,
              "E_kin": 364.05939964292014,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5745910449875,
              "E_x": -7.130165627372981,
              "E_xc": -22.12736707594001,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5745910449875,
            "energy_update": 0.000506458073232352,
            "mo_residual": 0.019653543108014405,
            "wall_time": 50.418660518
          },
          {
            "energy_terms": {
              "E_ee": 208.63506857461942,
              "E_eext": 0.0,
              "E_el": -403.8327343512643,
              "E_en": -947.1654043040579,
              "E_kin": 363.9511046991024,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5747017731052,
              "E_x": -7.129334060468581,
              "E_xc": -22.124169260459656,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5747017731052,
            "energy_update": 0.0001107281176473407,
            "mo_residual": 0.0067859691516417376,
            "wall_time": 52.396137991
          },
          {
            "energy_terms": {
              "E_ee": 208.62647319466853,
              "E_eext": 0.0,
              "E_el": -403.83277418749736,
              "E_en": -947.0756872930943,
              "E_kin": 363.8684396589236,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5747416093382,
              "E_x": -7.12912584536826,
              "E_xc": -22.12287390262691,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5747416093382,
            "energy_update": 3.98362330429336e-05,
            "mo_residual": 0.005688140437589139,
            "wall_time": 53.535888418
          },
          {
            "energy_terms": {
              "E_ee": 208.63024258844652,
              "E_eext": 0.0,
              "E_el": -403.8327820055483,
              "E_en": -947.0817695953879,
              "E_kin": 363.8713347681344,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5747494273892,
              "E_x": -7.1293063527925815,
              "E_xc": -22.12328341394879,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5747494273892,
            "energy_update": 7.81805096039534e-06,
            "mo_residual": 0.0025860999315262447,
            "wall_time": 53.877654696
          },
          {
            "energy_terms": {
              "E_ee": 208.6284642166044,
              "E_eext": 0.0,
              "E_el": -403.8327857274639,
              "E_en": -947.0855268186742,
              "E_kin": 363.8768835961993,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.57475314930474,
              "E_x": -7.129326284780402,
              "E_xc": -22.12328043681305,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.57475314930474,
            "energy_update": 3.7219155615275668e-06,
            "mo_residual": 0.0018928576468000807,
            "wall_time": 53.802332382
          },
          {
            "energy_terms": {
              "E_ee": 208.62535420626003,
              "E_eext": 0.0,
              "E_el": -403.8327874385555,
              "E_en": -947.0825287728752,
              "E_kin": 363.87675197149156,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.57475486039635,
              "E_x": -7.129275038471332,
              "E_xc": -22.123089804960514,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.57475486039635,
            "energy_update": 1.7110916132878629e-06,
            "mo_residual": 0.0015645290439173082,
            "wall_time": 54.935464541
          },
          {
            "energy_terms": {
              "E_ee": 208.62684201476287,
              "E_eext": 0.0,
              "E_el": -403.83278832286226,
              "E_en": -947.0904967116118,
              "E_kin": 363.88346684758125,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5747557447031,
              "E_x": -7.129325044349588,
              "E_xc": -22.123275429244988,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5747557447031,
            "energy_update": 8.843067575980967e-07,
            "mo_residual": 0.0011477667658905562,
            "wall_time": 56.101179073
          },
          {
            "energy_terms": {
              "E_ee": 208.62873884763573,
              "E_eext": 0.0,
              "E_el": -403.8327886076722,
              "E_en": -947.0960466882576,
              "E_kin": 363.8873184285471,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5747560295131,
              "E_x": -7.129362421426691,
              "E_xc": -22.12343677417069,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5747560295131,
            "energy_update": 2.848099711627583e-07,
            "mo_residual": 0.0005185394395581946,
            "wall_time": 55.679856008
          },
          {
            "energy_terms": {
              "E_ee": 208.62861657276278,
              "E_eext": 0.0,
              "E_el": -403.8327886819706,
              "E_en": -947.0935321715111,
              "E_kin": 363.88485832612645,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.57475610381147,
              "E_x": -7.129342537480359,
              "E_xc": -22.123388871868293,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.57475610381147,
            "energy_update": 7.429838433381519e-08,
            "mo_residual": 0.0002164037071819312,
            "wall_time": 57.108091707
          },
          {
            "energy_terms": {
              "E_ee": 208.62856557153745,
              "E_eext": 0.0,
              "E_el": -403.8327887010207,
              "E_en": -947.0924845456389,
              "E_kin": 363.88383341367864,
              "E_next": 0.0,
              "E_nn": 39.25803257815915,
              "E_nuc": 39.25803257815915,
              "E_tot": -364.5747561228616,
              "E_x": -7.129334736281606,
              "E_xc": -22.123368404316384,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -364.5747561228616,
            "energy_update": 1.9050105493079172e-08,
            "mo_residual": 9.028615269633616e-05,
            "wall_time": 65.955077204
          }
        ],
        "wall_time": 862.536238241
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
