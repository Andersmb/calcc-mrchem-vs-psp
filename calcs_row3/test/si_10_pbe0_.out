                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :        0.000000        0.000000       -1.035388
    1       O            :        0.000000        0.000000        1.817531
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              11
 Beta electrons          :                                              11
 Total electrons         :                                              22
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999996472665e-01
    1    1    a          :                              9.999999922747e-01
    2    1    a          :                              9.999999996096e-01
    3    1    a          :                              9.999999992041e-01
    4    1    a          :                              9.999999925102e-01
    5    1    a          :                              1.000000000034e+00
    6    1    a          :                              9.999999289951e-01
    7    1    a          :                              1.000000016459e+00
    8    1    a          :                              9.999999888173e-01
    9    1    a          :                              9.999999850548e-01
   10    1    a          :                              9.999999971019e-01
   11    1    b          :                              1.000000002214e+00
   12    1    b          :                              9.999999922625e-01
   13    1    b          :                              9.999999996183e-01
   14    1    b          :                              9.999999988761e-01
   15    1    b          :                              9.999997141157e-01
   16    1    b          :                              1.000000000034e+00
   17    1    b          :                              1.000001710361e+00
   18    1    b          :                              1.000000026134e+00
   19    1    b          :                              9.999999682004e-01
   20    1    b          :                              9.999999850875e-01
   21    1    b          :                              9.999999988278e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                362.175353534671
 E-N energy              :            (au)               -944.223430482622
 Coulomb energy          :            (au)                209.771362065500
 Exchange energy         :            (au)                 -7.182434072155
 X-C energy              :            (au)                -22.250995400310
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 39.258032578159
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -401.710144354916
 Nuclear energy          :            (au)                 39.258032578159
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.624521117768e+02
                         :      (kcal/mol)             -2.274421321990e+05
                         :        (kJ/mol)             -9.516178817381e+05
                         :            (eV)             -9.862823975743e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -65.929070405027
    1    1    a          :            (au)                -18.611055286891
    2    1    a          :            (au)                 -5.412481186973
    3    1    a          :            (au)                 -3.747345270510
    4    1    a          :            (au)                 -3.746923235712
    5    1    a          :            (au)                 -3.741017261572
    6    1    a          :            (au)                 -0.786074947507
    7    1    a          :            (au)                 -0.414663823339
    8    1    a          :            (au)                 -0.143109303162
    9    1    a          :            (au)                 -0.133401770375
   10    1    a          :            (au)                 -0.126484774676
   11    1    b          :            (au)                -65.929286900855
   12    1    b          :            (au)                -18.611056334492
   13    1    b          :            (au)                 -5.412482825521
   14    1    b          :            (au)                 -3.747345685113
   15    1    b          :            (au)                 -3.746923849914
   16    1    b          :            (au)                 -3.741019412652
   17    1    b          :            (au)                 -0.786075158153
   18    1    b          :            (au)                 -0.414664279873
   19    1    b          :            (au)                 -0.143108663704
   20    1    b          :            (au)                 -0.133385887428
   21    1    b          :            (au)                 -0.126485291244
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -205.583461554694
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.690416e+00        -362.452111776757         -3.624521e+02
    1          4.640213e-01        -364.431223068072         -1.979111e+00
    2          2.759387e-01        -364.544475595413         -1.132525e-01
    3          1.626300e-01        -364.561074309451         -1.659871e-02
    4          7.538805e-02        -364.571827871381         -1.075356e-02
    5          3.161746e-02        -364.574084586914         -2.256716e-03
    6          1.965354e-02        -364.574591044988         -5.064581e-04
    7          6.785969e-03        -364.574701773105         -1.107281e-04
    8          5.688140e-03        -364.574741609338         -3.983623e-05
    9          2.586100e-03        -364.574749427389         -7.818051e-06
   10          1.892858e-03        -364.574753149305         -3.721916e-06
   11          1.564529e-03        -364.574754860396         -1.711092e-06
   12          1.147767e-03        -364.574755744703         -8.843068e-07
   13          5.185394e-04        -364.574756029513         -2.848100e-07
   14          2.164037e-04        -364.574756103811         -7.429838e-08
   15          9.028615e-05        -364.574756122862         -1.905011e-08
---------------------------------------------------------------------------
                      SCF converged in 15 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Si            :        0.000000        0.000000       -1.035388
    1       O            :        0.000000        0.000000        1.817531
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                363.883833413679
 E-N energy              :            (au)               -947.092484545639
 Coulomb energy          :            (au)                208.628565571537
 Exchange energy         :            (au)                 -7.129334736282
 X-C energy              :            (au)                -22.123368404316
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 39.258032578159
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -403.832788701021
 Nuclear energy          :            (au)                 39.258032578159
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.645747561229e+02
                         :      (kcal/mol)             -2.287741116255e+05
                         :        (kJ/mol)             -9.571908836622e+05
                         :            (eV)             -9.920584068369e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -66.281245265048
    1    1    a          :            (au)                -19.192113319102
    2    1    a          :            (au)                 -5.364888196107
    3    1    a          :            (au)                 -3.699858054628
    4    1    a          :            (au)                 -3.699857413722
    5    1    a          :            (au)                 -3.693112926219
    6    1    a          :            (au)                 -0.969410212384
    7    1    a          :            (au)                 -0.453510289840
    8    1    a          :            (au)                 -0.349122155737
    9    1    a          :            (au)                 -0.349121060650
   10    1    a          :            (au)                 -0.324377258720
   11    1    b          :            (au)                -66.281245275080
   12    1    b          :            (au)                -19.192113300664
   13    1    b          :            (au)                 -5.364888177470
   14    1    b          :            (au)                 -3.699858050610
   15    1    b          :            (au)                 -3.699857432677
   16    1    b          :            (au)                 -3.693112922876
   17    1    b          :            (au)                 -0.969410180003
   18    1    b          :            (au)                 -0.453510291117
   19    1    b          :            (au)                 -0.349122222584
   20    1    b          :            (au)                 -0.349120917747
   21    1    b          :            (au)                 -0.324377246391
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -208.753232169377
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000001        0.000006       -1.283358
 Magnitude               :            (au)                        1.283358
                         :         (Debye)                        3.261970
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.044808
 Magnitude               :            (au)                        0.044808
                         :         (Debye)                        0.113891
---------------------------------------------------------------------------
 Total vector            :        0.000001        0.000006       -1.238550
 Magnitude               :            (au)                        1.238550
                         :         (Debye)                        3.148079
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 15m 23s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
